Workbooks/Resource Insights/AzureDataExplorerCluster/ResourceInsights.workbook

```diff
@@ -310,262 +1301,355 @@
-        ]
-      },
-      "conditionalVisibility": {
-        "parameterName": "tab",
-        "comparison": "isEqualTo",
-        "value": "overview"
-      },
-      "name": "Overview"
-    },
-    {
-      "type": 12,
-      "content": {
-        "version": "NotebookGroup/1.0",
-        "groupType": "editable",
-        "items": [
-          {
-            "type": 10,
-            "content": {
-              "chartId": "workbook9b15f2b0-76b4-4914-bee8-6c938f3e783c",
-              "version": "MetricsItem/2.0",
-              "size": 4,
-              "chartType": -1,
-              "metricScope": 0,
-              "resourceIds": [
-                "{Resource}"
-              ],
-              "timeContext": {
-                "durationMs": 0
-              },
-              "timeContextFromParameter": "TimeRange",
-              "resourceType": "microsoft.kusto/clusters",
-              "resourceParameter": "Resource",
-              "metrics": [
-                {
-                  "namespace": "microsoft.kusto/clusters",
-                  "metric": "microsoft.kusto/clusters-Export health and performance-ContinuousExportNumOfRecordsExported",
-                  "aggregation": 1
-                },
-                {
-                  "namespace": "microsoft.kusto/clusters",
-                  "metric": "microsoft.kusto/clusters-Export health and performance-ContinuousExportMaxLatenessMinutes",
-                  "aggregation": 3
-                },
-                {
-                  "namespace": "microsoft.kusto/clusters",
-                  "metric": "microsoft.kusto/clusters-Export health and performance-ContinuousExportPendingCount",
-                  "aggregation": 3
-                },
-                {
-                  "namespace": "microsoft.kusto/clusters",
-                  "metric": "microsoft.kusto/clusters-Export health and performance-ContinuousExportResult",
-                  "aggregation": 7
-                },
-                {
-                  "namespace": "microsoft.kusto/clusters",
-                  "metric": "microsoft.kusto/clusters-Export health and performance-ExportUtilization",
-                  "aggregation": 3
-                }
-              ],
-              "gridFormatType": 1,
-              "tileSettings": {
-                "titleContent": {
-                  "columnMatch": "Metric",
-                  "formatter": 13,
-                  "formatOptions": {
-                    "linkTarget": null,
-                    "showIcon": false
-                  }
-                },
-                "leftContent": {
-                  "columnMatch": "Value",
-                  "formatter": 12,
-                  "formatOptions": {
-                    "palette": "auto"
+                "formatters": [
+                  {
+                    "columnMatch": "$gen_group",
+                    "formatter": 13,
+                    "formatOptions": {
+                      "linkTarget": "Resource",
```

Comment:
in these groupings, we should have a subtarget to 'insights'. For now it will redirect to the over view, but later when we plug the 'at-resource' template in then it will go to the 'insights' menu item. Please change this for all of the $gen_group links

```
                  {
                    "columnMatch": "$gen_group",
                    "formatter": 13,
                    "formatOptions": {
                      "linkTarget": "Resource",
                      "subTarget": "insights",
                      "showIcon": true
                    }
                  },
```

Reply:
this grouping wasn't being used in the workbook. I removed it.